functions-iterators/starter.js

```diff
@@ -29,11 +29,26 @@ This will help us track your progress and it will give you practice writing good
 
 // PART 0: Just DO it!!
 
+function justDoIt(){
+  return "JUST DO IT!";
```

Comment:
in this part, you need to return the name you passed to the function with the string so your first mistake is not having a parameter in your function `function justDoIt(name){}` the second mistake return the name  with the string `return name + "JUST DO IT!"`